core/src/util/jobQueue.cpp

```diff
@@ -0,0 +1,40 @@
+#include "jobQueue.h"
+#include <mutex>
+
+namespace Tangram {
+
+std::unordered_map<JobQueue::ThreadId, std::vector<JobQueue::Job>> JobQueue::jobsForThreads;
+std::mutex jobMutex;
```

Comment:
Though both are fine here.
But, in a general practice, global scoped mutex are used to sync global scoped variables.
Its a better practice to have a static mutex class member to sync a static class member variable.